### PR TITLE
FISH-5722 document concurrency config in payara-resources.xml

### PR DESCRIPTION
**Pull request by aubi**

### CODE REVIEW

File: enterprise/docs/antora.yml [hunk 1]
```diff
@@ -6,10 +6,10 @@ nav:
   - modules/ROOT/nav.adoc
 asciidoc:
   attributes:
-    currentVersion: '5.36.0'
+    currentVersion: '5.39.0'
     glassfishVersion: '5'
-    payaraWebDtd: 'https://raw.githubusercontent.com/payara/Payara-Enterprise-Documentation/master/docs/modules/ROOT/pages/schemas/payara-web-app_4.dtd'
-    payaraResourcesDtd: 'https://raw.githubusercontent.com/payara/Payara-Enterprise-Documentation/master/docs/modules/ROOT/pages/schemas/payara-resources_1_6.dtd'
+    payaraWebDtd: 'https://raw.githubusercontent.com/payara/Payara-Documentation/master/docs/modules/ROOT/pages/Appendix/Schemas/payara-web-app_4.dtd'
+    payaraResourcesDtd: 'https://raw.githubusercontent.com/payara/Payara-Documentation/master/docs/modules/ROOT/pages/Appendix/Schemas/payara-resources_1_6.dtd'
```
Comment (Pandrex247) [inline enterprise/docs/antora.yml:12]: Should this now not be 1_7?

**Review by Pandrex247 — CHANGES_REQUESTED** [review]

File: enterprise/docs/antora.yml (hunk 1 above)
Comment (Pandrex247) [reply enterprise/docs/antora.yml:12]: it's being added in this PR

Comment (Pandrex247) [reply enterprise/docs/antora.yml:12]:
```suggestion
    payaraResourcesDtd: 'https://raw.githubusercontent.com/payara/Payara-Documentation/master/docs/modules/ROOT/pages/Appendix/Schemas/payara-resources_1_7.dtd'
```

Comment (aubi) [reply enterprise/docs/antora.yml:12]: Copy&paste error :-)

**Comment by aubi** [comment]
@fturizo Fabio, it is a good idea. I was thinking about it -- it will need to show, how to use ManagedExecutorService, ManagedScheduledExecutorService and ManagedThreadFactory first. Then how to define them in GUI and then how to do the same in payara-resources.xml.
Let's do a Jira ticket for it. It's a bit more work.

**Review by Pandrex247 — APPROVED** [review]

**Comment by Pandrex247** [comment]
Documentation will also need republishing once this is merged, this is for the release which went out today and I believe the 5.40.0 docs have been published

**Comment by AlanRoth** [comment]
> Documentation will also need republishing once this is merged, this is for the release which went out today and I believe the 5.40.0 docs have been published

I'll sort it

**Pull request merged**